lib/linter.js

```diff
@@ -145,17 +145,15 @@ function parseJsonConfig(string, location) {
  */
 function parseListConfig(string) {
     const items = {};
+    const reg = /[\w-/]+/g;
+    const rs = string.match(reg);
```

Comment:
Can you give this variable a more descriptive name? The name `rs` doesn't communicate what the variable is doing very well.

Maybe something like `elementMatches` would be better.